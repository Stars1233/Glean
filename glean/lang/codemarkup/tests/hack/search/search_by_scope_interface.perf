{
  "@generated": null,
  "facts_searched": {
    "hack.DeclarationSpan.6": 1,
    "hack.Name.6": 2,
    "hack.NamespaceQName.6": 1,
    "hack.QName.6": 5,
    "hack.SearchClassConstByName.6": 1
  },
  "full_scans": [ ]
}