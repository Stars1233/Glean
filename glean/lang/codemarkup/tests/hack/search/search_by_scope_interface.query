query: codemarkup.search.SearchByScope { name = "JAZZ", scope = ["SourceInterface"], searchcase = Sensitive, language = Hack }
transform: [normord, []]
perf: true
