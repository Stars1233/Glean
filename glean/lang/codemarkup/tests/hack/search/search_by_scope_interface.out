[
  "@generated",
  {
    "key": {
      "entity": {
        "hack": {
          "decl": {
            "classConst": {
              "key": {
                "container": {
                  "interface_": {
                    "key": {
                      "name": {
                        "key": { "name": { "key": "SourceInterface" } }
                      }
                    }
                  }
                },
                "name": { "key": "JAZZ" }
              }
            }
          }
        }
      },
      "kind": 15,
      "language": 5,
      "location": {
        "file": { "key": "www/SourceInterface.php" },
        "location": { "span": { "length": 24, "start": 111 } },
        "name": "JAZZ"
      },
      "name": "JAZZ",
      "scope": [ "SourceInterface" ],
      "searchcase": 0
    }
  }
]